### ISSUE TITLE
How does this compare to Sentry KMP?

### ISSUE THREAD

**Issue by PaulWoitaschek** [rickclephas/NSExceptionKt#22]
Hey Rick!

Thanks for building this library <3 

I wonder: How does this compare to the sentry KMP library?
https://github.com/getsentry/sentry-kotlin-multiplatform

If we're using sentry: Does your integration do more than the official one?

**Comment by rickclephas** [comment]
Hi! So I am not really up-to-date on the state of the Sentry KMP SDK.
However the major difference is that NSExceptionKt only solves the unhandled Kotlin exceptions issue.
The Sentry SDK would (eventually) provide a lot more functionality that you can already find in their native SDKs.
For the unhandled Kotlin exceptions their SDK is using the [same logic](https://github.com/getsentry/sentry-kotlin-multiplatform/blob/fe4b42f45bcbcd0bb54f529ebca90b0c774eab53/sentry-kotlin-multiplatform/src/commonAppleMain/kotlin/io/sentry/kotlin/multiplatform/nsexception/NSException.kt) as NSExceptionKt.

TLDR: if possible I would go with their official SDK 😇.

**Comment by buenaflor** [comment]
Hey! The SDK includes Rick's implementation for fixing unhandled exceptions and adds support for using Sentry directly in KMP (not only kotlin/native).

Interesting integrations other than error monitoring include for example screenshots, view hierarchies, etc 
We are also currently working on adding support for performance monitoring

Feel free to open a discussion or issue in the repo if you have any other question 😄

**Comment by rickclephas** [comment]
Note: as of [v1.0.0-BETA-1](https://github.com/rickclephas/NSExceptionKt/releases/tag/v1.0.0-BETA-1) the Sentry implementation is deprecated in favour of the official [Sentry Kotlin Multiplatform SDK](https://github.com/getsentry/sentry-kotlin-multiplatform).
